spellcheck/platform/linux/enchant.cpp

```diff
@@ -0,0 +1,253 @@
+/* enchant
```

Comment:
If we have platform dependent code files, we should name them in the same way as it is done in the [TDesktop](https://github.com/telegramdesktop/tdesktop/tree/dev/Telegram/SourceFiles/platform/linux) —  for one-platform files we have to add the platform name as a prefix.
So probably it should be `spellcheck/platform/linux/linux_enchant.cpp` and `spellcheck/platform/linux/linux_enchant.h`.

Reply:
I renamed the files to `enchant_linux.cpp` and `enchant_linux.h` respectively according to a form that Telegram Desktop already uses.